content/blog/2023-06-21-comparison-of-ethereum-classic-and-fiat-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+Fiat money is decided by a committee of central bankers that meets every few months. They evaluate the state of the economy and of government finances, and they decide whether to expand the base money stock or whether to contract it. They also decide at what level they will set interest rates. Therefore, fiat money is not algorithmic, but managed by technocrats.
+
+## Capped Supply
+
+ETC has a capped supply of its coin. This means that it’s known what is the total amount of ETC that will ever exist.
+
+Given Ethereum Classic’s frequency of payments to miners, the number of coins it issues per block, and its discount schedule, it will have a maximum supply of 210,700,000 ETC.
+
+Fiat money does not have a cap and is decided subjectively by government officials. As said before, they meet frequently and decide how much base money they will print or redeem in the market. This is a recipe for biased manipulation of the supply as the government itself is the largest consumer of newly minted money, and this has led to ballooning deficits, unmanageable national debt, inflation, and chronic depreciation of the value of fiat money around the world.
+
+![Federal Reserve's assets, which are analogous to base money.](./2.png)
+
+The above chart is a perfect example of how the issuance of base money has grown uncontrollably in the last 20 years in the US. In 2002 the total base money, as represented by the central bank’s assets, was $712 billion. In 2023, it is more than $8.38 trillion. This is an average of more than 13% money issuance per year.
+
+## Block Time
+
+Ethereum Classic’s block time is 13 seconds per block. This means that ETC miners create 6,646 blocks per day.
+
+The block time of a blockchain is important because it establishes how fast the coins will be created, thus regulates the supply, providing certainty to the economy. 
+
+By modulating the block times and the number of coins created per block a blockchain can control the monetary policy in time and supply levels forever.
+
+Fiat money is created whenever the government wishes and there is no guidance or certainty as to the base money supply projection other than the brief communications of the central bank every few months.
+
+## Eras
+
+Ethereum Classic’s issuance is divided in eras. Fiat money issuance is unpredictable and decided arbitrarily by a committee whenever they wish.
+
+Eras in ETC are measured in blocks in which the payments to miners, thus the issuance of coins, will be the same for each era.
+
+Ethereum Classic’s eras last 5 million blocks which, at 13 seconds per block, turns out to be more or less 2 years per era.
+
+After each era ETC discounts the payment to miners.
+
+## Discount Per Eras
+
+In Ethereum Classic, the discount per era is 20%. 
+This means that since genesis, ETC has paid miners 5 ETC per block in the first era, then 4 ETC, then 3.20, and now is paying 2.56 which is the current era.
```

Comment:
Line 92: Insert blank line; Rewrite for clarity
```suggestion

This means that since genesis, ETC has paid miners 5 ETC per block in the first era, then 4 ETC, then 3.20, and now pays 2.56 per block in the current era.
```